themes/bootstrap3/less/mixins/dropdowns.less

```diff
@@ -0,0 +1,22 @@
+// Dropdowns Mixin
+
+// Mixin for dropdown link styles:
+// This mixin uses the styles from lines 65ff in vendor/bootstrap/dropdowns.less
```

Comment:
I've changed this. "ff." was supposed to read "pp" or "et seq" since I assumed the vendor files don't change, I thought it safe to refer the exact line(s).

The other changes you noticed are unrelated but I guess it might be worth investigating where they come from (I suspect missing `!default` declarations and such).